redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -197,12 +197,17 @@ def _check_db(self):
         conn = self._get_conn(self.instance)
         # Ping the database for info, and track the latency.
         # Process the service check: the check passes if we can connect to Redis
-        start = time.time()
+        if PY2:
+            start = time.time()
+        else:
+            start = time.process_time()
         tags = list(self.tags)
         try:
             info = conn.info()
-            latency_ms = round_value((time.time() - start) * 1000, 2)
-
+            if PY2:
+                latency_ms = round_value((time.time() - start) * 1000, 2)
+            else:
+                latency_ms = round_value((time.process_time() - start) * 1000, 2)
```

Comment:
nit: Just to be extra explicit on what the actual difference between the two branches is maybe you can do 
```suggestion
            if PY2:
                cur_time = time.time()
            else:
                cur_time = time.process_time()
            latency_ms = round_value((cur_time - start) * 1000, 2)
```

or even 

```
            cur_time = time.time() if PY2 else time.process_time()
            latency_ms = round_value((cur_time - start) * 1000, 2)
```

Reply:
Done